hasher-matcher-actioner/webapp/src/utils/MediaUtils.tsx

```diff
@@ -74,16 +70,12 @@ export const BlurImage = BlurMedia(
  * Show a blurred preview of a video at {src}.
  */
 export const BlurVideo = BlurMedia(
-  (handleMouseLeave, handleMouseEnter, blur, src): JSX.Element => (
-    <div className="video-preview-container">
-      {/* eslint-disable jsx-a11y/media-has-caption */}
-      <video
-        onMouseLeave={handleMouseLeave}
```

Comment:
no more hover option?

Reply:
Yeah, I found myself revealing too many images as I scanned through the grid view on the bank.

Hypothesis: User is more likely to not want to view the content. If they do, they'll click the button. Pattern matching from "<internal_system>" UI.